test/junit/scala/collection/mutable/ArraySortingTest.scala

```diff
@@ -25,7 +25,7 @@ class ArraySortingTest {
     java.util.Arrays.sort(test)
     scala.util.Sorting.quickSort(cant)(CanOrder)
     assert( test(6) == 1 )
-    assert( (test,cant).zipped.forall(_ == _.i) )
+    assert( test.toIterable.lazyZip(cant).forall(_ == _.i) )
```

Comment:
We should add `lazyZip` to `ArrayOps` so that users don’t have to convert it to an `Iterable`. We could do the same with `StringOps`, btw. I’ve created https://github.com/scala/bug/issues/11063 for that. You don’t have to do it in that PR.